src/app/execution-context.js

```diff
@@ -87,32 +86,24 @@ function ExecutionContext () {
     return vm
   }
 
-  this.setEndPointUrl = function (url) {
-    $web3endpoint.val(url)
-  }
-
-  this.setContext = function (context) {
+  this.setContext = function (context, endPointUrl) {
     executionContext = context
-    executionContextChange(context)
-  }
-
-  var $web3endpoint = $('#web3Endpoint')
-
-  if (web3.providers && web3.currentProvider instanceof web3.providers.IpcProvider) {
-    $web3endpoint.val('ipc')
+    executionContextChange(context, endPointUrl)
   }
 
-  function executionContextChange (context) {
-    if (context === 'web3' && !confirm('Are you sure you want to connect to a local ethereum node?')) {
+  function executionContextChange (context, endPointUrl) {
+    if (context === 'web3' && !confirm('Are you sure you want to connect to an external ethereum node?')) {
```

Comment:
External sounds like it will not be on the same computer. Perhaps just say `to an Ethereum node`?